Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -31,19 +42,17 @@ public ContactsCenter(string peopleFile)
         public List<Person> Filter(Predicate<Person> predicate)
```

Comment:
Can return IEnumerable<Person> instead, because it allows assigning it to more generalized forms.